src/main/java/flaxbeard/steamcraft/client/ClientProxy.java

```diff
@@ -35,27 +33,24 @@
 import net.minecraft.entity.EntityLivingBase;
 import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.item.Item;
-import net.minecraft.util.DamageSource;
 import net.minecraft.util.ResourceLocation;
 import net.minecraft.world.World;
 import net.minecraft.world.WorldSettings.GameType;
 import net.minecraftforge.client.MinecraftForgeClient;
-import net.minecraftforge.event.entity.living.LivingAttackEvent;
 import org.lwjgl.input.Keyboard;
 
+import java.util.ArrayList;
+
 
 public class ClientProxy extends CommonProxy {
     public static final ResourceLocation villagerTexture = new ResourceLocation("steamcraft:textures/models/villager.png");
-    public KeyBinding zoomKey = new KeyBinding("Zoom using monocle", Keyboard.KEY_Z, "key.categories.misc");
+    public static ArrayList<KeyBinding> keyBindings = new ArrayList<>();
```

Comment:
Make this a hashmap so it is easier to reason about the code. Such as when below you get keybinding 0 you should probably get keybinding monocle. That would mean for `keybinding.foreach(keyBindings.forEach(ClientRegistry::registerKeyBinding)` you would need something like `keyBindings.values.forEach(ClientRegistry::registerKeyBinding)` instead (IIRC HashMaps have a value only method)